modules/database/src/admin/schema.admin.ts

```diff
@@ -35,7 +35,11 @@ export class SchemaAdmin {
         undefined,
         undefined,
         'name parentSchema fields extensions modelOptions ownerModule collectionName',
-      );
+        { updatedAt: 1 },
```

Comment:
This won't fix import requests hanging due to yet-to-be-imported related schemas being sorted incorrectly.
We should figure out why the pending schema functionality won't work on the import side.